armada/armada.py

```diff
@@ -24,16 +24,14 @@ def __init__(self, args):
 
         self.tiller = Tiller()
 
-
     def find_release_chart(self, known_releases, name):
         '''
         Find a release given a list of known_releases and a release name
         '''
-        for chart_name, version, chart, values in known_releases:
+        for chart_name, _, chart, values in known_releases:
```

Comment:
What was the purpose of removing `version` here?  I'm a little skeptical of replacing it with the "dont care" variable `_` here.